docs/userguide/async.md

```diff
@@ -0,0 +1,74 @@
+# Asynchronous Workflows
+
+Asynchronous workflows describe situations where the loop between measurements and
+recommendations is more complex and needs to incorporate various other aspects. These
```

Comment:
I think better to refer to "measurement" and "recommendation" in terms of the *processes*, not the resulting quantities
```suggestion
Asynchronous workflows describe situations where the loop between measurement and
recommendation is more complex and needs to incorporate various other aspects. These
```